docs/todos.md

```diff
@@ -44,28 +44,44 @@ todos can be created with optional due dates. These due dates allow for todos to
 
 Due dates can be configured in multiple ways, but all specify integers for `warn` and `error` to signify the number of days from the todo created date to decay the severity.
 
+Due dates can be configured on a per-rule basis as well with the `daysToDecayByRule` option.  See examples below.
+
 :bulb: Both `warn` and `error` are optional. The value for `warn` should be lower than the value of `error`.
 
-1. Via package.json configuration
+1. Via `package.json`
 
    ```json
    {
      "lintTodo": {
-       "decayDays": {
+       "daysToDecay": {
          "warn": 5,
          "error": 10
        }
      }
    }
    ```
 
-1. Via environment variables
+2. Via `.lint-todorc.js`
```

Comment:
These should all stay as `1.`. That is the markdown notation for a numbered list, they will be numbered automatically. If you manually number them, then the numbers have to be updated by hand every time items are added/removed/reordered.

Reply:
Correct!